script.js

```diff
@@ -232,3 +235,13 @@ checkAddOns.forEach((addOn, pos) => {
     addOnCards[pos].classList.toggle('add-ons__card--active')
   })
 })
+
+//Retrieve data
+function retrieveData() {
+  if (globalSectionIndex === 0) {
+    data.name = nameInput.value
+    data.email = emailInput.value
+    data.phone = phoneInput.value
```

Comment:
if you are going to set the values this way you can remove all of the update methods